src/main/java/site/coach_coach/coach_coach_server/coach/service/CoachService.java

```diff
@@ -209,6 +209,14 @@ public CoachListResponse getAllCoaches(User user, int page, String sports, Strin
 		return new CoachListResponse(coaches, (int)coachesPage.getTotalElements(), page);
 	}
 
+	@Transactional
+	public void cancelLikeCoach(Long userId, Long coachId) {
+		coachRepository.findById(coachId)
+			.orElseThrow(() -> new NotFoundCoachException(ErrorMessage.NOT_FOUND_COACH));
+
+		userCoachLikeRepository.deleteByUser_UserIdAndCoach_CoachId(userId, coachId);
```

Comment:
이거 실행 전에 좋아요 관계가 존재하는지 확인 한 후에 삭제하는 것도 좋을 것 같아요! 안정성이 높아질 것 같습니다

```suggestion
		if (userCoachLikeRepository.existsByUser_UserIdAndCoach_CoachId(userId, coachId)) {
        userCoachLikeRepository.deleteByUser_UserIdAndCoach_CoachId(userId, coachId);
    } else {
        // throw new로 예외 처리 
    }
```